Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -16,19 +23,22 @@ def csv_file_to_indicator_list(file_path, col_num, starting_row, auto_detect, de
     with open(file_path) as csv_file:
         file_reader = csv.reader(csv_file)
         for row in file_reader:
-            if line_index >= starting_row:
+            if line_index >= starting_row + offset and len(row) != 0:
                 indicator = row[col_num]
 
                 indicator_type = detect_type(indicator)
 
-                if indicator_type is None:
-                    continue
+                if not auto_detect:
+                    indicator_type = default_type
 
                 if type_col:
                     indicator_type = row[int(type_col) - 1]
```

Comment:
are we sure that this access will not be out of range?

Reply:
yes I explicitly stated in the argument description that the type_col should start from 1:
![image](https://user-images.githubusercontent.com/50295826/79737296-c87d0b00-8303-11ea-87c4-f03a90ada9c1.png)